packages/plexus/src/Digraph/SvgEdge.tsx

```diff
@@ -62,15 +64,31 @@ export default class SvgEdge<U = {}> extends React.PureComponent<TProps<U>> {
       },
       getProps(setOnEdge, layoutEdge, renderUtils)
     );
+
+    const [startX, startY] = pathPoints[0];
+    const [endX, endY] = pathPoints[pathPoints.length - 1];
+
+    const xOffset = (label?.length ?? 0) * 5;
+    const labelX = (startX + endX) / 2;
+    const labelY = (startY + endY) / 2;
```

Comment:
```suggestion
    const computeLabelCoords = () => {
      const [startX, startY] = pathPoints[0];
      const [endX, endY] = pathPoints[pathPoints.length - 1];
      const xOffset = (label?.length ?? 0) * 5;
      const labelX = (startX + endX) / 2 - xOffset;
      const labelY = (startY + endY) / 2;
      return { labelX, labelY };
    };
    const { labelX, labelY } = computeLabelCoords();
```
this way you do not pollute the scope with temporary variables that are only relevant for calculation

Reply:
That's a good suggestion, more memory efficient